examples/MBPOL/splitting/TS/input.xml

```diff
@@ -5,7 +5,7 @@
         <trajectory stride='1' filename='pos'>  positions{angstrom}  </trajectory>
    </output>
    <total_steps>400       </total_steps>
-   <ffsocket name="aims" mode="unix" >
+   <ffsocket name="qtip4pf" mode="unix" >
```

Comment:
Why is this inside "MBPOL" splitting folder and apparently uses as a driver qtip4p?

Reply:
P.S.: Other files in the folder are the same.

Reply:
The automatic tester looks for the name variable in ffsocket to assign a driver model.
If it recognizes it uses the corresponding model. Otherwise uses gas.

These are old examples that I used do check the code against JOR results using MBPOL.

Reply:
We need to discuss this infrastructure.